anime_downloader/sites/init.py

```diff
@@ -14,6 +14,7 @@
     ('animeflix', 'animeflix', 'AnimeFlix'),
     ('darkanime', 'darkanime', 'DarkAnime'),
     ('dubbedanime', 'dubbedanime', 'Dubbedanime'),
+    ('animechameleon', 'gurminderboparai', 'AnimeChameleon'),
```

Comment:
Would personally prefer a shorter name, having to do --provider gunminderboparaitigatagitviga for every command is tiresome.